python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_row_count_postprocessing.py

```diff
@@ -123,3 +123,79 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
 
         # assert we have warning message in logs
         assert "An error occurred while fetching row count for " in caplog.text
+
+
+@pytest.fixture(name="test_jaffle_shop_manifest_snowflake")
+def test_jaffle_shop_manifest_snowflake_fixture() -> Dict[str, Any]:
+    return _create_dbt_invocation(test_jaffle_shop_path, target="snowflake").get_artifact(
+        "manifest.json"
+    )
+
+
+@pytest.mark.prodsnowflake
```

Comment:
If possible, we should parameterize the mark. The same test should be executed on `duckdb`, `snowflake`, and `bigquery`.

Also, let's just make the mark `snowflake` and `bigquery`, rather than `prodsnowflake`, `prodbigquery`.

Reply:
Good call, collapsed into one test